package.json

```diff
@@ -7,58 +7,58 @@
-    "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-flowtype": "^2.50.3",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-prettier": "^2.7.0",
+    "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-relay": "^0.0.19",
     "flow-bin": "^0.56.0",
-    "gatsby": "^2.0.0",
-    "gatsby-plugin-catch-links": "^2.0.0",
-    "gatsby-plugin-feed": "^2.0.0",
-    "gatsby-plugin-glamor": "^2.0.0",
-    "gatsby-plugin-google-analytics": "^2.0.0",
-    "gatsby-plugin-manifest": "^2.0.0",
-    "gatsby-plugin-netlify": "^2.0.0",
-    "gatsby-plugin-nprogress": "^2.0.0",
-    "gatsby-plugin-react-helmet": "^3.0.0",
-    "gatsby-plugin-sharp": "^2.4.12",
-    "gatsby-plugin-twitter": "^2.0.0",
-    "gatsby-remark-code-repls": "^2.0.0",
-    "gatsby-remark-copy-linked-files": "^2.0.0",
-    "gatsby-remark-embed-snippet": "^3.0.0",
+    "gatsby": "^2.20.3",
+    "gatsby-plugin-catch-links": "^2.2.1",
+    "gatsby-plugin-feed": "^2.4.1",
+    "gatsby-plugin-glamor": "^2.2.1",
+    "gatsby-plugin-google-analytics": "^2.2.1",
+    "gatsby-plugin-manifest": "^2.3.2",
+    "gatsby-plugin-netlify": "^2.2.1",
+    "gatsby-plugin-nprogress": "^2.2.1",
+    "gatsby-plugin-react-helmet": "^3.2.1",
+    "gatsby-plugin-sharp": "^2.5.2",
+    "gatsby-plugin-twitter": "^2.2.1",
+    "gatsby-remark-code-repls": "^2.2.0",
+    "gatsby-remark-copy-linked-files": "^2.2.1",
+    "gatsby-remark-embed-snippet": "^3.2.4",
     "gatsby-remark-external-links": "^0.0.4",
-    "gatsby-remark-images": "^2.0.0",
-    "gatsby-remark-prismjs": "^3.0.2",
-    "gatsby-remark-responsive-iframe": "^2.0.0",
-    "gatsby-remark-smartypants": "^2.0.0",
-    "gatsby-source-filesystem": "^2.0.0",
-    "gatsby-transformer-remark": "^2.0.0",
-    "gatsby-transformer-sharp": "^2.3.18",
-    "github-slugger": "^1.2.1",
+    "gatsby-remark-images": "^3.2.1",
+    "gatsby-remark-prismjs": "^3.4.1",
+    "gatsby-remark-responsive-iframe": "^2.3.1",
+    "gatsby-remark-smartypants": "^2.2.1",
+    "gatsby-source-filesystem": "^2.2.2",
+    "gatsby-transformer-remark": "^2.7.1",
+    "gatsby-transformer-sharp": "^2.4.2",
+    "github-slugger": "^1.3.0",
     "glamor": "^2.20.40",
     "hex2rgba": "^0.0.1",
-    "mdast-util-to-string": "^1.0.5",
-    "normalize.css": "^8.0.0",
-    "prettier": "^1.7.4",
-    "prismjs": "^1.15.0",
+    "mdast-util-to-string": "^1.1.0",
+    "normalize.css": "^8.0.1",
+    "prettier": "^1.19.1",
+    "prismjs": "^1.19.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-helmet": "^5.2.0",
-    "react-live": "1.8.0-0",
-    "remarkable": "^1.7.1",
-    "request-promise": "^4.2.2",
-    "rimraf": "^2.6.1",
-    "slugify": "^1.2.1",
-    "unist-util-visit": "^1.1.3"
+    "react-helmet": "^5.2.1",
+    "react-live": "1.12.0",
+    "remarkable": "^1.7.4",
+    "request-promise": "^4.2.5",
+    "rimraf": "^2.7.1",
+    "slugify": "^1.4.0",
+    "unist-util-visit": "^1.4.1"
   },
   "engines": {
-    "node": "12.x.x",
+    "node": ">8.4.0",
```

Comment:
?

Reply:
Oops that's something left from rebasing 😅